README.md

```diff
@@ -213,13 +207,70 @@ Currently supported built-in compilable objects are:
 
 `Code` objects can be concatenated together by using the `+` operator.
 
+### Verifying the accounts' Post State
+
+The state of the accounts after all blocks/transactions have been executed is
+the way of verifying that the execution client actually behaves like the test
+expects.
+
+During their filling process, all tests automatically verify that the accounts
+specified in their `post` property actually match what was returned by the
+transition tool.
+
+Within the `post` dictionary object, an account address can be:
+- `None`: The account will not be checked for absence or existance in the
+  result returned by the transition tool.
+- `Account` object: The test expects that this account exist and also has
+  properties equal to the properties specified by the `Account` object.
+- `Account.NONEXISTENT`: The test expects that this account does not exist in
+  the result returned by the transition tool, and if the account exists,
+  it results in error.
+  E.g. when the transaction creating a contract is expected to fail and the
+  test wants to verify that the address where the contract was supposed to be
+  created is indeed empty.
+
+### The `Account` object
+
+The `Account` object is used to specify the properties of an account to be
+verified in the post state.
+
+It has the following Python representation.
+```
+class Account:
```

Comment:
I wish there were a nice way to provide a ref in markdown to a code object and have it fetch it so that you don't need to worry about keeping the two in sync.

Reply:
Yeah, I think it might be better to remove it altogether and point to the file containing the definition.